test/common/network/udp_fuzz.cc

```diff
@@ -0,0 +1,166 @@
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "common/api/os_sys_calls_impl.h"
+#include "common/network/address_impl.h"
+#include "common/network/socket_option_factory.h"
+#include "common/network/socket_option_impl.h"
+#include "common/network/udp_listener_impl.h"
+#include "common/network/udp_packet_writer_handler_impl.h"
+#include "common/network/utility.h"
+
+#include "test/common/network/udp_listener_impl_test_base.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/mocks/api/mocks.h"
+#include "test/mocks/network/mocks.h"
+#include "test/mocks/server/mocks.h"
+#include "test/test_common/environment.h"
+#include "test/test_common/network_utility.h"
+#include "test/test_common/threadsafe_singleton_injector.h"
+#include "test/test_common/utility.h"
+
+namespace Envoy {
+//	namespace Network {
+namespace {
+
+class UdpFuzz;
+
+class FuzzUdpListenerCallbacks : public Network::UdpListenerCallbacks {
+public:
+  FuzzUdpListenerCallbacks(UdpFuzz* upf) : my_upf(upf) {}
+  ~FuzzUdpListenerCallbacks() override = default;
+  UdpFuzz* my_upf;
```

Comment:
nit: this can be private, name with a trailing underscore per style (https://google.github.io/styleguide/cppguide.html)